components/Cards/SchoolCard/BackSchoolCard.js

```diff
@@ -27,7 +27,7 @@ function BackSchoolCard({ cardFlipCallback, locations, logoSource, schoolName })
       <hr className={styles.divider} />
       <ul className={styles.locations}>
         {locations.map(location => (
-          <li key={`${location}`}>
+          <li key={`${JSON.stringify(location)}`}>
```

Comment:
Instead of this, you should be able to just do `location.name` since we only have one card per school. It's unique, but small.

Reply:
I think it needs to be something unique to id each location. It could be address1 + address2 from location obj. Es-lint did not like index from map